apps/prairielearn/src/lib/question-render.ts

```diff
@@ -724,24 +794,19 @@ export async function renderPanelsForSubmission({
       // is disabled, so it does not need to be replaced.
       if (variant.instance_question_id == null || next_instance_question.id == null) return;
 
-      /** @type {{can_view: boolean} | null} */
-      let groupRolePermissions = null;
-      /** @type {string} */
+      let groupRolePermissions: { can_view: boolean } | null = null;
```

Comment:
Nit: there is already a `group_role_permissions` variable in this function, and in more than one occasion I confused this one with that one. This one is intended to represent the permissions of the next question. Maybe rename it to `nextQuestionGroupRolePermission` or something like that to avoid this confusion?

Reply:
Good call; done in 5260fb8.